config/custom-environment-variables.yaml

```diff
@@ -12,6 +12,7 @@ services:
     clientID: OAUTH_CLIENT_ID
     clientSecret: OAUTH_CLIENT_SECRET
     callbackURL: OAUTH_CLIENT_REDIRECT
+    userDashboard : IDAM_USER_DASHBOARD
```

Comment:
I'd change variable name to userDashboardURL to be clearer (i.e to denote what the string is)

Reply:
fixed